### ISSUE TITLE
I want to use it by ubuntu without graphic interface

### ISSUE THREAD

**Issue by xh789123** [chidiwilliams/buzz#863]
hello ,I want to ask. Can I use buzz without graphic interface. how to do that

**Comment by raivisdejus** [comment]
User interface is the main feature of the Buzz that makes it easier to use for new users or for users not familiar with command line interface. 

Buzz has CLI options that you can use, they are described here https://chidiwilliams.github.io/buzz/docs/cli
This will still open the full app with graphical interface so may not work on server that does not have any graphical subsystems installed on it. 

To sum up you sort of can use Buzz without a graphical interface, but such use case most likely is part of some broader process you may get better results by using the transcriptions directly. For example https://github.com/openai/whisper

```
whisper japanese.wav --language Japanese --model medium
```

**Comment by xh789123** [comment]
thank you for the answer. Hope you get rich soon

---- Replied Message ----
| From | Raivis ***@***.***> |
| Date | 07/30/2024 23:54 |
| To | chidiwilliams/buzz ***@***.***> |
| Cc | xh789123 ***@***.***>,
Author ***@***.***> |
| Subject | Re: [chidiwilliams/buzz] I want to use it by ubuntu without graphic interface (Issue #863) |

User interface is the main feature of the Buzz that makes it easier to use for new users or for users not familiar with command line interface.

Buzz has CLI options that you can use, they are described here https://chidiwilliams.github.io/buzz/docs/cli
This will still open the full app with graphical interface so may not work on server that does not have any graphical subsystems installed on it.

To sum up you sort of can use Buzz without a graphical interface, but such use case most likely is part of some broader process you may get better results by using the transcriptions directly. For example https://github.com/openai/whisper

whisper japanese.wav --language Japanese --model medium
